build.sbt

```diff
@@ -29,10 +31,17 @@ lazy val release = Seq[ReleaseStep](
   pushChanges
 )
 
-lazy val commonSettings = Seq(
+inThisBuild(Seq(
   organization := "com.gu",
   scalaVersion := "2.12.10",
-  resolvers ++= Seq(Resolver.sonatypeRepo("releases"), Resolver.bintrayRepo("guardian", "ophan")),
+  dependencyStats / aggregate := false,
+  dependencyTree / aggregate := false,
+  // https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html
+  updateOptions := updateOptions.value.withCachedResolution(true),
+  resolvers ++= Seq(Resolver.sonatypeRepo("releases")), // libraries that haven't yet synced to maven central
+))
+
+lazy val releaseSettings = Seq(
```

Comment:
separated release settings out, because they only apply to a minority of projects.